news/10031.bugfix.rst

```diff
@@ -0,0 +1 @@
+Remove support for ``setup.cfg``-only projects. See the discussion in https://github.com/pypa/pip/pull/9945.
```

Comment:
This isn't a good changelog entry IMO. Suggested rephrasing:

> Require setup.cfg-only projects to be built via PEP 517, by requiring an explicit dependency on setuptools declared in pyproject.toml.

Reply:
Is this a bugfix or a removal ? I don't think this was announced but...
Also does this need to go in yet another 21.1 bugfix release, before people start relying on it ?

Reply:
> This isn't a good changelog entry IMO. Suggested rephrasing:

Done.

> Is this a bugfix or a removal ? I don't think this was announced but...

Bugfix. This was not announced and added in a PR that had nothing to do with the behavior change, it was only supposed to improve the clarity of the error.

>  Also does this need to go in yet another 21.1 bugfix release, before people start relying on it ?

I'd really appreciate it if you could put it in a 21.1 bugfix release, or people might start relying on it. It would be particularly bad if a distro like Debian decides to pick 21.1 for their next stable release.

Reply:
Alrighty, I'll do a 21.1.3.